v10.12 Changelog (#8300)
mattermost/docs: source/product-overview/mattermost-v10-changelog.md

**Release day: 2025-09-16**

```{Important}
If you upgrade from a release earlier than v10.10, please read the other [Important Upgrade Notes](https://docs.mattermost.com/administration-guide/upgrade/important-upgrade-notes.html).
```

### Compatibility
 - Updated minimum Edge and Chrome versions to 138+.

### Improvements

#### User Interface (UI)
 - Improved screen reader readouts and accessibility of various autocomplete components.

#### Administration
 - Added an endpoint to check for upgradability. Also, a correct error message is now shown and the “Upgrade Server and Start trial” button is disabled if an upgrade is not possible.

### Bug Fixes
 - Fixed an issue where pasting a link when a text was selected didn't format the selected text as a Markdown link when editing the message.
 - Fixed an issue where emoji picker items were being selected by pressing the space bar.
 - Fixed a potential crash in ``UpdatePost``.
 - Fixed 403 errors that occurred when loading custom profile attributes for unauthenticated users.
 - Fixed an issue where the **Back** button was not showing on the desktop external login redirect page.

>  - Fixed an issue were unread messages from muted channels were shown in the favicon/desktop app.

Reviewer: ```suggestion
 - Fixed an issue where unread messages from muted channels were shown in the favicon/desktop app.
```

```{include} common-esr-support-upgrade.md
```

(release-v10.12-feature-release)=
## Release v10.12 - [Feature Release](https://docs.mattermost.com/product-overview/release-policy.html#release-types)

**Release day: 2025-09-16**

```{Important}

> If you upgrade from a release earlier than v10.10, please read the other [Important Upgrade Notes](https://docs.mattermost.com/administration-guide/upgrade/important-upgrade-notes.html).

Reviewer 2: Should this be v10.11?

Author: There were no important upgrade notes in v10.11.

```

### Compatibility
 - Updated minimum Edge and Chrome versions to 138+.

### Improvements

#### User Interface (UI)
 - Improved screen reader readouts and accessibility of various autocomplete components.

>  - Improved screen reader readouts of various autocomplete components.

Reviewer 2: Some duplication between these 2 bullets

#### Administration

>  - Added an endpoint to check for upgradability, and correct error message and disable button are now shown if upgrade is not possible.

Reviewer 2: What disable button is shown if an upgrade isn't possible?

### Bug Fixes

>  - Fixed an issue where pasting a link when a text was selected didn't format the selected text as a markdown link when editing the post.

Reviewer 2: ```suggestion
 - Fixed an issue where pasting a link when a text was selected didn't format the selected text as a Markdown link when editing the message.
```

 - Fixed an issue where emoji picker items were being selected by pressing the space bar.
 - Fixed a potential crash in ``UpdatePost``.
 - Fixed 403 errors that occurred when loading custom profile attributes for unauthenticated users.

>  - Fixed an issue where the back button was not showing on the desktop external login redirect page.

Reviewer 2: ```suggestion
 - Fixed an issue where the **Back** button was not showing on the desktop external login redirect page.
```

>  - Fixed an issue were unreads from muted channels were shown in the favicon/desktop app.

Reviewer 2: ```suggestion
 - Fixed an issue were unread messages from muted channels were shown in the favicon/desktop app.
```

 - Fixed an issue where extra content was not accounted for in the focus order.

>  - Fixed an issue where results were not allowed to be read on filter without a search term.

Reviewer 2: Is this an accessibility issue? I'm not sure I understand what's been fixed.

>  - Fixed an issue where some webhooks would not show.

Reviewer 2: Is there more detail about this one and where the webhooks weren't showing up?
